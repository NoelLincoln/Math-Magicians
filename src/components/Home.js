import style from '../styles/home.css';

const Home = () => (
  <section className={style.section}>
    <h4>Welcome to our page</h4>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum rerum
      dolores ea quos exercitationem libero et eveniet porro est similique. Eos
      repudiandae dolore nam veniam itaque voluptas, incidunt necessitatibus
      maxime.
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum rerum
      dolores ea quos exercitationem libero et eveniet porro est similique. Eos
      repudiandae dolore nam veniam itaque voluptas, incidunt necessitatibus
      maxime.
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum rerum
      dolores ea quos exercitationem libero et eveniet porro est similique. Eos
      repudiandae dolore nam veniam itaque voluptas, incidunt necessitatibus
      maxime.
    </p>
  </section>
);

export default Home;
