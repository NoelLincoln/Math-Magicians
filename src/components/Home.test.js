import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import Home from './Home';

test('Home component snapshot', () => {
  const { asFragment } = render(<Home />);
  expect(asFragment()).toMatchSnapshot();
});
