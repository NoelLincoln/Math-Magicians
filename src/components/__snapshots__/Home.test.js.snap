// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component snapshot 1`] = `
<DocumentFragment>
  <section>
    <h4>
      Welcome to our page
    </h4>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum rerum dolores ea quos exercitationem libero et eveniet porro est similique. Eos repudiandae dolore nam veniam itaque voluptas, incidunt necessitatibus maxime.
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum rerum dolores ea quos exercitationem libero et eveniet porro est similique. Eos repudiandae dolore nam veniam itaque voluptas, incidunt necessitatibus maxime.
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum rerum dolores ea quos exercitationem libero et eveniet porro est similique. Eos repudiandae dolore nam veniam itaque voluptas, incidunt necessitatibus maxime.
    </p>
  </section>
</DocumentFragment>
`;
